### PR TITLE
Adding inspect

### PR DESCRIPTION
**Pull request by geramirez**
I never used the inspector would this work? @kiranjthomas

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "nodemonConfig": {
-    "exec": "npm run build_client && npm run start",
+    "exec": "npm run build_client && npm run start -- --inspect",
```
Comment (kiranjthomas) [inline package.json]: Adding --inspect to the start script worked for me:

    "start": "node --inspect index.js",


Here are some instructions on how to use the inspector in Chrome:

1. Open `chrome://inspect` in Chrome
1. Click `Configure`
1. Add  `localhost:9229` and click Done (when node is started with --inspect, a Node.js process listens for a debugging client at port 9229)
1. Click `Open dedicated DevTools for Node`
1. Add a debugger statement (e.g. add debugger to /status endpoint in index.js)
1. Run `npm run startDev` and debug!

Comment (geramirez) [reply package.json]: thanks that's super useful.

**Comment by geramirez** [comment]
@kiranjthomas okay I fixed it and adding your documentation.

**Review by kiranjthomas — APPROVED** [review]
LGTM!

**Pull request merged**